src/utils/message-utils.ts

```diff
@@ -137,13 +137,13 @@ export const catchError = (diagnostics: d.Diagnostic[], err: Error | null | unde
   };
 
   if (isString(msg)) {
-    diagnostic.messageText = msg;
+    diagnostic.messageText = msg.length ? msg : 'UNKNOWN ERROR';
   } else if (err != null) {
     if (err.stack != null) {
       diagnostic.messageText = err.stack.toString();
     } else {
       if (err.message != null) {
-        diagnostic.messageText = err.message.toString();
+        diagnostic.messageText = err.message.length ? err.message.toString() : 'UNKNOWN ERROR';
```

Comment:
Presumably it's safe to remove `toString()` since `err.message` is a string?
```suggestion
        diagnostic.messageText = err.message.length ? err.message : 'UNKNOWN ERROR';
```

Reply:
Good point, removing in 3fa068d91..e131ccaf5